python/lsst/faro/base/MatchedCatalogBase.py

```diff
@@ -35,6 +35,30 @@ class MatchedBaseTaskConnections(pipeBase.PipelineTaskConnections,
         dimensions=("skymap",),
     )
 
+    def __init__(self, *, config=None):
+        """Customize connection for the astrometric calibrations
+
+        Parameters
+        ----------
+        config : `MatchedBaseTaskConfig`
+            A config for `MatchedBaseTask` or one of its subclasses
+        """
+        super().__init__(config=config)
+        if config and config.wcsDimensions != self.astrom_calibs.dimensions:
+            # Hack, this is the only way to get a connection without fixed dims
+            # Inspired by:
+            # https://github.com/lsst/verify/blob/master/python/lsst/verify/tasks/metadataMetricTask.py#L74-L87
```

Comment:
Might be better to use a hash rather than `master` here so the reference doesn't move underneath you.